distribution/sdk/sample-cometd/cometd-web-example/src/main/webapp/js/application.js

```diff
@@ -65,7 +65,7 @@
             // Add Notification Subscription
             cometd.subscribe('/ddf/notifications/**', function(message) {
               console.log("Got a message!");
```

Comment:
I believe `console.log` would not be recommended in DDF UIs, so it would probably good to make an explicit comment somewhere about that. This has been here for the sample and should be fine. Maybe somebody from the UI team can provide better guidance:@codice/ui